packages/bot-web-ui/src/utils/__tests__/bot-notifications.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import { journalError } from '../bot-notifications';
+import { localize } from '@deriv/translations';
+
+describe('journalError function', () => {
+    it('Should return the correct error object', () => {
+        const mockOnClick = jest.fn();
+        const error = journalError(mockOnClick);
+
+        expect(error.key).toBe('bot_error');
+        expect(error.header).toBe(localize('The bot encountered an error while running.'));
```

Comment:
@maryia-matskevich-deriv why we need to localize in the test file?
are we changing the language anywhere?
```suggestion
        expect(error.header).toBe('The bot encountered an error while running.');
```

Reply:
fixed✅